ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -255,8 +259,8 @@
 
 #### BGP Neighbors
 
-| Neighbor | Remote AS | VRF | Shutdown | Send-community | Maximum-routes | Allowas-in | BFD | RIB Pre-Policy Retain | Route-Reflector Client | Passive |
-| -------- | --------- | --- | -------- | -------------- | -------------- | ---------- | --- | --------------------- | ---------------------- | ------- |
+| Neighbor | Remote AS | VRF | Shutdown | Send-community | Maximum-routes | Allowas-in | BFD | BFD Intervals | RIB Pre-Policy Retain | Route-Reflector Client | Passive |
```

Comment:
I am wondering if we could not instead put the timers in the existing BFD column when they are defined - thoughts  @Claus Holbech ?

Reply:
discussed - let's just add the timers in parenthesis when configured

Reply:
Yes sounded good to me as well.
Have done the change, thanks